sktime/forecasting/compose/tests/test_pipeline.py

```diff
@@ -478,10 +478,14 @@ def test_forecastx_logic():
     assert np.allclose(y_pred, y_pred_manual)
 
 
+@pytest.mark.skipif(
```

Comment:
can we:

* leave the `NaiveForecaster` as one option, as it ignores `X`
* instead add a test, but use an exogenous forecaster that does not require soft deps, e.g., `YfromX`.

Reply:
Addressed.